src/components/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+
+import ComponentRegister, { LinkComponent } from "../ComponentRegister";
+
+import "lbh-frontend/lbh/core/_links.scss";
+
+/**
+ * Generates a Link component that's a wrapper for the Link attribute of
+ * ComponentRegister
```

Comment:
I think this is more than a wrapper. It's a component to link to both internal and external pages that can be customized but `ComponentRegister`. The default is fine as `"a"` and we shouldn't make people go looking for `ComponentRegister` to understand what this does.